carbon/assetlist.json

```diff
@@ -201,6 +297,18 @@
         "name": "Circle USD",
         "display": "usdc",
         "symbol": "USDC",
+        "traces": [{
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "USDC",
```

Comment:
the base denom is the contract address
0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48